waspc/data/Generator/templates/react-app/src/entities/index.ts

```diff
@@ -0,0 +1,18 @@
+{{={= =}=}}
```

Comment:
If this is the same as the one in server (they are _nearly_ identical) can it go in shared instead perhaps?

Reply:
Hm, the thing is, we don't really have a shared templates folder. Sure, we allow users to put their stuff in `src/shared`, but that's just for the users.

We could implement a shared code mechanism (at will probably definitely need to at some point). Did you have any ideas on how this could look like?

Reply:
Oh yeah, true it's just for users. Ok, let's not overcomplicate it then and just punt on this one! 👍🏻